metadata/metadata.ts

```diff
@@ -214,6 +214,18 @@ export const OPEN_SEA_ITEMS: Record<InventoryItemName, Metadata> = {
       { trait_type: "Tradable", value: "No" },
     ],
   },
+  "Banana Plant": {
+    name: "Banana Plant",
+    description:
+      "A plant used to grow bananas.\n\nYou can buy banana plants in game at the market.",
+    decimals: 18,
+    external_url: "https://docs.sunflower-land.com/getting-started/about",
+    image_url: "../public/erc1155/images/120.png",
+    attributes: [
+      { trait_type: "Purpose", value: "Seed" },
```

Comment:
Does this `Purpose` imply Kuebiko should impact the price?

Reply:
@brian-man Yeah right now Kuebiko will impact the price.